invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides

```diff
@@ -188,6 +188,17 @@ code {
   border: 0 !important;
 }
 
+.media-inline-block > *{
+  display: inline-block;
+}
+
+.computer-flex-header{
```

Comment:
why do we need `computer-` (device) specific classes? the idea was that with Media component we will not need them anymore

Reply:
This is for a Jinja template, I could use the "computer only" class but I would create a lot of code duplication.